packages/app-static/README.md

```diff
@@ -5,6 +5,42 @@ title: KeystoneJS Static File App
 
 # KeystoneJS Static File App
 
-```DOCS_TODO
-TODO
+A Keystone App to serve static files such as images, CSS and JavaScript with support for client side routing.
+
+## Usage
+
+`index.js`
+
+```js
+const { Keystone } = require('@keystone-alpha/keystone');
+const { GraphQLApp } = require('@keystone-alpha/app-graphql');
+const { AdminUIApp } = require('@keystone-alpha/app-admin-ui');
+const { StaticApp } = require('@keystone-alpha/app-static');
+
+module.exports = {
+  new Keystone(),
+  apps: [
+    new GraphQLApp(),
+    new AdminUIApp(),
+    new StaticApp({
+      path: '/',
+      src: pathModule.join(__dirname, 'build'),
```

Comment:
Remove `pathModule.join(...)` and replace with just `src`. I think this is a simpler usage example.